scripts/config-table-generator.py

```diff
@@ -0,0 +1,83 @@
+import re
```

Comment:
Where is this used?

Reply:
I wrote this to automatically generate the markdown tables for the [Node Configuration Options](https://github.com/waku-org/docs.waku.org/blob/add-nwaku-guides/docs/guides/reference/node-config-options.md) page from the [wakunode2 config file](https://github.com/waku-org/nwaku/blob/master/apps/wakunode2/config.nim)